jackson-simple/src/test/java/com/baeldung/jackson/annotation/JacksonAnnotationUnitTest.java

```diff
@@ -95,6 +95,13 @@ public void whenSerializingUsingJsonValue_thenCorrect() throws IOException {
         assertThat(enumAsString, is("1609.34"));
     }
 
+    @Test
+    public void whenSerializingFieldUsingJsonValue_thenCorrect() throws IOException {
+        final String enumAsString = new ObjectMapper().writeValueAsString(PriorityEnum.HIGH);
+
+        assertThat(enumAsString, is("3"));
```

Comment:
In favor of not using deprecated methods we can change this asserts to:
        `assertEquals("3", enumAsString);`

Reply:
yes I thought about that, but I said it's better to keep it consistent with other methods